charts/opentelemetry-demo/values.yaml

```diff
@@ -348,12 +378,17 @@ components:
       - name: QUOTE_SERVICE_PORT
         value: "8080"
     podAnnotations: {}
-    #  instrumentation.opentelemetry.io/inject-sdk: "true"
+    resources:
+      limits:
+        memory: 30Mi
 
 
 opentelemetry-collector:
   nameOverride: otelcol
   mode: deployment
+  resources:
+    limits:
```

Comment:
This will result in a collector with 

```yaml
resources:
  limits:
    cpu: 256m
    memory: 100Mi
```

Do you want to decrease the cpu as well?

Reply:
When I did the long-running tests I only looked at memory, though given we don't use any processors, we should likely lower CPU as well, I just don't have any data to support what it should be.